values.yaml

```diff
@@ -53,7 +53,7 @@ global:
   prometheusPostgresExporterEnabled: false
 
   # Enables blackbox exporter
-  blackboxExporterEnabled: true
+  blackboxExporterEnabled: false
```

Comment:
we should default to what enterprises mostly do, in this case most enterprises do not have use for bbe